contracts/farm-bot.sol

```diff
@@ -168,37 +195,47 @@ contract FarmBot is Owned, FarmbotERC20 {
         stakingRewards.getReward();
 
         // compute bounty for the caller
-        uint256 _tokenBalance = rewardsToken.balanceOf(address(this));
-        if (_tokenBalance == 0) {
-            return;
-        }
-        uint256 _bountyAmount;
+        uint256[] memory _tokenBalances = new uint256[](rewardsTokens.length);
+	TokenAmount[] memory _interestAccrued = new TokenAmount[](rewardsTokens.length);
+
+	for (uint i=0; i< rewardsTokens.length; i++) {
+	    _tokenBalances[i] = rewardsTokens[i].balanceOf(address(this));
+	    _interestAccrued[i] = TokenAmount(rewardsTokens[i], _tokenBalances[i]);
+	}
+
+	uint256[] memory _bountyAmounts = new uint256[](rewardsTokens.length);
         {  // block is to prevent 'stack too deep' compilation error.
-            TokenAmount[] memory _interestAccrued = new TokenAmount[](1);
-            _interestAccrued[0] = TokenAmount(rewardsToken, _tokenBalance);
-            _bountyAmount = revoBounty.calculateFeeBounty(_interestAccrued)[0].amount;
+	    TokenAmount[] memory _feeBounties = revoBounty.calculateFeeBounty(_interestAccrued);
+	    for (uint i=0; i < _feeBounties.length; i++) {
+		_bountyAmounts[i] = _feeBounties[i].amount;
+		assert(_bountyAmounts[i] <= maxFeeNumerator * _tokenBalances[i] / maxFeeDenominator);
```

Comment:
```suggestion
		require(_bountyAmounts[i] <= maxFeeNumerator * _tokenBalances[i] / maxFeeDenominator, "Bounty amount too high");
```

Reply:
I realize I put this `assert` here originally, but I forgot about `require`, which is more readable and debuggable imo

Reply:
yeah, i also forgot about `require` lol, good catch.